src/mainboard/clevo/adl-p/variants/nv40pz/gpio.c

```diff
@@ -0,0 +1,881 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+
+#include <mainboard/gpio.h>
+#include <soc/gpio.h>
+
+/* Pad configuration was generated automatically using intelp2m utility */
+static const struct pad_config gpio_table[] = {
+
+	/* ------- GPIO Community 0 ------- */
+
+	/* ------- GPIO Group GPP_B ------- */
+
+	/* GPP_B0 - CORE_VID0 */
+	PAD_CFG_NF(GPP_B0, NONE, DEEP, NF1),
+
+	/* GPP_B1 - CORE_VID1 */
+	PAD_CFG_NF(GPP_B1, NONE, DEEP, NF1),
+
+	/* GPP_B2 - GPIO */
+	PAD_CFG_GPO(GPP_B2, 0, DEEP),
+
+	/* GPP_B3 - GPIO */
+	PAD_CFG_GPI_TRIG_OWN(GPP_B3, NONE, DEEP, OFF, ACPI),
+
+	/* GPP_B4 - GPIO */
+	PAD_CFG_GPI_TRIG_OWN(GPP_B4, NONE, DEEP, OFF, ACPI),
+
+	/* GPP_B5 - I2C2_SDA */
+	PAD_CFG_NF(GPP_B5, NONE, DEEP, NF2),
+
+	/* GPP_B6 - I2C2_SCL */
+	PAD_CFG_NF(GPP_B6, NONE, DEEP, NF2),
+
+	/* GPP_B7 - GPIO */
+	PAD_CFG_GPO(GPP_B7, 0, DEEP),
+
+	/* GPP_B8 - GPIO */
+	PAD_CFG_GPO(GPP_B8, 0, DEEP),
+
+	/* GPP_B9 - GPIO */
+	PAD_CFG_GPO(GPP_B9, 0, DEEP),
+
+	/* GPP_B10 - GPIO */
+	PAD_CFG_GPO(GPP_B10, 0, DEEP),
```

Comment:
These do not seem to be present on schematics, shouldn't they be NC instead?

Reply:
Set GPP_B10 as NC